<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Mouli" failures="1" tests="3" name="Personal Loan" time="0.137" errors="0" timestamp="2021-04-01T23:30:01 IST">
  <testcase classname="TestPackage.firstLesson" name="Demo" time="0.051">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestPackage.firstLesson.Demo(firstLesson.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="TestPackage.firstLesson" name="SecondDemo" time="0.002"/>
  <testcase classname="TestPackage.secondLesson" name="Demo" time="0.005"/>
</testsuite> <!-- Personal Loan -->
